### DISCUSSION TITLE
Incompressible aerodynamics robustness

### DISCUSSION THREAD

**Discussion by justAPhDStudent** [exadg/exadg#99] [Performance] [answered]
I was just wondering the robustness of the exaDG solvers for the incompressible aerodynamics problems, with Reynolds number in the  range of millions. And, are the programs robust enough to perform with real world parameters, or do we need to non-dimensionalize the quantities. 

Thanks in advance

**Comment by kronbichler** [comment, marked as answer]
In general, the methods are robust for very large Reynolds numbers in terms of the iLES (implicit LES) features of the DG discretizations. We ran the 3D Taylor-Green vortex in the inviscid limit (Re -> inf), so at least in the interior of the domain there is a reasonable behavior. The big question for these cases are boundary layers and there high-orders are no silver bullet, as one still needs to refine quite a lot towards the boundary to resolve the gradients. That in turn leads to high aspect ratios and higher costs in the associated solvers. I think the highest numbers we've been considering are in the range of Re~100k. We did some work with wall modeling in the past, https://onlinelibrary.wiley.com/doi/10.1002/fld.4712 , but that code is not included in ExaDG (yet) because it would need more research to make it generally useful. (But it could be an interesting project.)

Regarding real world parameters: In principle we tried hard to make the solver scale-invariant, so you can feed it both non-dimensional numbers and actual units. So I would consider anything that does not run well a bug, and we should work around that. Are you interested along trying to set up some specific case where we could test things?

**Comment by justAPhDStudent** [comment]
Thanks for the reply. Currently the test cases I deal with are all external aerodynamics problems where the Reynolds number exceed the limit currently done with exaDG. Once I have some cases wotrh communicating, surely I will. It would be great to have an example with a gmsh (or any other 3rdparty) interface, & also the boundary condition options in the prm files; so that we can easily test things on the go, to see if there is some hope with these libraries. Thank you !
